doc/blocks.xml

```diff
@@ -173,3 +173,27 @@ gap> DegreeOfBlocks(blocks);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="CanonicalBlocks">
+  <ManSection>
+    <Attr Name = "CanonicalBlocks" Arg = "blocks"/>
+    <Returns>Blocks of a bipartition.</Returns>
+    <Description>
+      For the blocks of a bipartition <A>blocks</A> the function
+      <C>CanonicalBlocks</C> returns a canonical represnetative of the
+      <A>blocks</A>.
+      <P/>
+
+      In particular, let <C>C</C> be a class of blocks such that for every pair
+      of elements <C>x</C> and <C>y</C> of <C>C</C> the number of signed, and
+      similarly unsigned, blocks of any given size in both <C>x</C> and
+      <C>y</C> are the same. Then <C>CanonicalTransformation</C> returns a
+      canonical represnetative of <C>C</C>.
```

Comment:
As I said below on the doc about transformations, perhaps end with a statement declaring precisely what the function does to the specific argument. i.e. returns the canonical rep of the class to which the arg belongs.